addons/event_sms/models/event_mail.py

```diff
@@ -64,15 +63,6 @@ def execute(self):
 
         return super(EventMailScheduler, self).execute()
 
-    @api.onchange('notification_type')
-    def set_template_ref_model(self):
```

Comment:
You kept the original method for mail, but removed the overrides ? As it is probably not used anymore, maybe we could remove it also. Or alternatively we keep the code (or part of it) for backward compatibility. But not sure this code would be really useful, especially it was recently introduced by Hale.

Reply:
Indeed 99% odds nobody relies on it (legal disclaimer: not backed by research)